backend/src/main/java/develup/submission/SubmissionApi.java

```diff
@@ -0,0 +1,27 @@
+package develup.submission;
+
+import java.net.URI;
+import develup.member.Member;
+import develup.support.ApiResponse;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+class SubmissionApi {
+
+    private final SubmissionService submissionService;
+
+    public SubmissionApi(SubmissionService submissionService) {
+        this.submissionService = submissionService;
+    }
+
+    @PostMapping("/submissions")
+    public ResponseEntity<ApiResponse<SubmissionResponse>> postSubmission(@RequestBody CreateSubmissionRequest request) {
+        Member member = new Member(1L);
+        SubmissionResponse newSubmission = submissionService.submit(member, request);
```

Comment:
[Comment]

newSubmission은 entity를 의미하는 것 같아서 submissionResponse나 response가 어떨까요? 🤫
 
```suggestion
        SubmissionResponse submissionResponse = submissionService.submit(member, request);
```

Reply:
response 좋네요! 반영했습니다. 😊